src/OrbitQt/ProfilingTargetDialog.cpp

```diff
@@ -265,13 +264,14 @@ void ProfilingTargetDialog::SetupStadiaStates() {
   // STATE s_s_process_selected
   state_stadia_process_selected_.addTransition(this, &ProfilingTargetDialog::NoProcessSelected,
                                                &state_stadia_no_process_selected_);
-  QObject::connect(&state_stadia_process_selected_, &QState::entered, [this] {
+  QObject::connect(&state_stadia_process_selected_, &QState::entered, this, [this] {
     CHECK(process_ != nullptr);
-    QString process_name = QString::fromStdString(process_->name());
     CHECK(ui_->stadiaWidget->GetSelectedInstance().has_value());
-    QString instance_name = ui_->stadiaWidget->GetSelectedInstance().value().display_name;
-    ui_->targetLabel->setText(QString("%1 @ %2").arg(process_name).arg(instance_name));
+    ui_->targetLabel->ChangeToStadiaTarget(*process_,
+                                           ui_->stadiaWidget->GetSelectedInstance().value());
   });
+  QObject::connect(&state_stadia_process_selected_, &QState::exited, this,
+                   [this] { ui_->targetLabel->Clear(); });
```

Comment:
why not
QObject::connect(&state_stadia_process_selected_, &QState::exited, ui_->targetLabel, &TargetLabel::Clear);?

Reply:
done